webapp/channels/NOTICE.txt

```diff
@@ -1455,41 +1481,6 @@ SOFTWARE.
 
 ---
 
-## emoji-datasource-apple
```

Comment:
we use this as well https://github.com/mattermost/mattermost/blob/release-9.0/webapp/channels/package.json#L145
:D

Reply:
Added it back :)